tests/Resources/ti.geolocation.addontest.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2011-Present by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* global Ti */
```

Comment:
<!--
  1 failure:  tests/Resources/t...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.geolocation.addontest.js;
  Line: 8;
-->
- :no_entry_sign: tests/Resources/ti.geolocation.addontest.js line 8 – 'Ti' is already defined as a built-in global variable. (no-redeclare)